demos/ripples.html

```diff
@@ -12,7 +12,7 @@
       <svg width="{{width}}" height="{{height}}">
         <rect x="0" y="0" width="{{width}}" height="{{height}}" fill="#1B1B1B" />
 
-        {{#each points.items key="key" as |point|}}
```

Comment:
Yeah, we used to have hax in glimmer to skip undefined values. Maybe we can subclass TestEnvironment, override iteratorFor and use the custom forEach in RingBuffer?